packages/table/src/cell/editableCell.tsx

```diff
@@ -57,6 +57,7 @@ export class EditableCell extends React.Component<IEditableCellProps, {}> {
                         className={"bp-table-editable-name"}
                         defaultValue={value}
                         intent={intent}
+                        minWidth={null}
```

Comment:
The min/max is set by the table's minColumnWidth/maxColumnWidth and is handled elsewhere. We don't want them to compete. I don't know how to unset a default other than passing null